.standard.yml

```diff
@@ -0,0 +1,5 @@
+ignore:
+  - 'lib/roadie/rails/mailer.rb':
+    - Style/ArgumentsForwarding 
+  - 'lib/roadie/rails/options.rb':
+    - Performance/StringIdentifierArgument
```

Comment:
Missing new line at the end again :P
I will add https://editorconfig.org and set `insert_final_newline`
You should setup your editor(s)/IDE(s) to follow https://editorconfig.org config if present (though I have never seen any project allowing missing new line
I will simply merge and fix it later

Reply:
Woops sorry!